tests/Functional/Schema/SQLiteSchemaManagerTest.php

```diff
@@ -214,15 +214,15 @@ public function testNonSimpleAlterTableCreatedFromDDL(): void
 
         $table1 = $schemaManager->introspectTable('nodes');
         $table2 = clone $table1;
-        $table2->addIndex(['name'], 'idx_name');
+        $table2->addIndex(['name'], 'idx_node_name');
```

Comment:
Apparently, index names in SQLite are global, so I renamed the one here and in the other test which now also runs on SQLite.